src/screens/home/components/epoch/index.test.tsx

```diff
@@ -11,17 +19,38 @@ const mockI18n = {
   lang: 'en',
 };
 jest.mock('next-translate/useTranslation', () => () => mockI18n);
+jest.mock('@components', () => ({
+  Box: (props) => <div id="box" {...props} />,
+}));
+
+const mockEpoch = jest.fn().mockResolvedValue({
+  data: {
+    average_time: '0.535677777',
+    slot: '420000',
+  },
+});
 
 // ==================================
 // unit tests
 // ==================================
 describe('screen: Home/Epoch', () => {
-  it('matches snapshot', () => {
+  it('matches snapshot', async () => {
+    const mockClient = createMockClient();
+
+    mockClient.setRequestHandler(
+      EpochDocument,
+      mockEpoch,
+    );
+
     const component = renderer.create(
-      <MockTheme>
-        <Epoch />
-      </MockTheme>,
+      <ApolloProvider client={mockClient}>
+        <MockTheme>
+          <Epoch />
+        </MockTheme>
+      </ApolloProvider>,
     );
+    await wait();
+
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
```

Comment:
I want to see better testing. Since you can mock the `average_time` and `slot` we should be able to check what the expected `epochTime` time is. 

Some reference:

https://github.com/forbole/big-dipper-2.0-cosmos/blob/chains/sifchain/src/components/msg/clp/add_liquidity/index.test.tsx